### ISSUE TITLE
setPedAnimation() messes up collisions of peds inside vehicles

### ISSUE THREAD

**Issue by Einheit-101** [multitheftauto/mtasa-blue#884]
**Describe the bug**
This bug seems to be as old as MTA, but i recently discovered this after investigating some bug reports of my server. People told me that some players in vehicles can not be killed and shots go right through them.
Specifically, i have some vehicles where i use setPedAnimation() to change the way how these peds are sitting in the vehicle. 

**To reproduce**
-Go to editor
-Create a bike (ID 509)
-Stand on top of the bike and execute:
srun veh = getPedContactElement(me) boi = createPed(0, getElementPosition(me)) warpPedIntoVehicle(boi, veh) setPedAnimation(boi, "ped", "idle_stance", -1, true, true, true, false)
-Try to shoot dat boi with a weapon, notice how you cant hit him

**Expected behaviour**
Shots should hit dat boi

**Multi Theft Auto v1.5.6-release-16632**

**Comment by StifflersMom** [comment]
YEAH! This affects realdriveby in many ways. We blocked realdriveby with bikes a few days before because more and more players reported "hacks" with players, which are immortal. Checking it, there were no "hacks" of course, they were using a weapon and realdriveby ...

**Comment by FileEX** [comment]
Cannot reproduce, seems to be fixed
